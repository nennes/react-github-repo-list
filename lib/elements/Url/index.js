import React from "react";
import PropTypes from "prop-types";

const Url = ({ url }) => {
  return (
    <div>
      {url}
    </div>
  );
};

Url.propTypes = {
  url: PropTypes.string.isRequired
};

export default Url;
