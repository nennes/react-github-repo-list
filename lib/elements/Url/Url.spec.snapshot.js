/* global test, expect */

import React from "react";
import Url from "./index";
import renderer from "react-test-renderer";

test("Url element renders correctly", () => {
  const element = renderer.create(<Url url="http://www.example.com" />);
  expect(element.toJSON()).toMatchSnapshot();
});
