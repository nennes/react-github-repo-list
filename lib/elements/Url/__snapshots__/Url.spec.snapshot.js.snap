// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Url element renders correctly 1`] = `
<div>
  http://www.example.com
</div>
`;
